GitHub-Org-Management-Policy.md

```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
 
 ### Owners
 
-TSC members and the Chair of the Community Committee are the only individuals granted
-Owner permissions within the Node.js GitHub Organization.
+As with other resources within the community, Org ownership will be
+granted in order to optimize for the conflicting requirements of:
+
+* Limiting access to reduce risk
+* Enabling individuals to move community work forward without undue delay
+
+When possible, automation and tools should be used to reduce the breadth of
+access that needs to be provided in order to enable individuals to move
+community work forward.  As these tools are developed and put in place we will
+reduce the groups to which org ownership is granted.
+
+At the current time, the following groups are granted access in order for them
```

Comment:
Nit: "At the current time" is superfluous. Take it out.

Reply:
Also consider removing "in order for them to facilitate community work.". The sentence should end with a colon as well. So all told, maybe this?:

> The following groups are granted Ownership permissions: